subsys/bluetooth/controller/ll_sw/ull_peripheral.c

```diff
@@ -160,7 +160,7 @@ void ull_periph_setup(struct node_rx_pdu *rx, struct node_rx_ftr *ftr,
 	if ((lll->data_chan_count < CHM_USED_COUNT_MIN) ||
 	    (lll->data_chan_hop < CHM_HOP_COUNT_MIN) ||
 	    (lll->data_chan_hop > CHM_HOP_COUNT_MAX) ||
-	    !lll->interval) {
+	    !IN_RANGE(lll->interval, BT_HCI_LE_INTERVAL_MIN, BT_HCI_LE_INTERVAL_MAX)) {
```

Comment:
Do you want to use the below define or a new define to allow proprietary connection intervals to establish connections?

https://github.com/zephyrproject-rtos/zephyr/blob/91b7204dc140e100041018cdbed22f0ecb9fe679/subsys/bluetooth/controller/ll_sw/ull_internal.h#L10-L20

```suggestion
	    !IN_RANGE(lll->interval, CONN_INTERVAL_MIN, BT_HCI_LE_INTERVAL_MAX)) {
```

Reply:
As far as I understand, ´CONFIG_BT_CTLR_USER_CPR_INTERVAL_MIN´ is not related to the interval used in CONNECT_IND. It is only used in the LLCP connection parameter request procedure, primarily to support Apple devices that do not fully comply with the Bluetooth specification. For CONNECT_IND (which is used to establish a connection), the interval must be at least ´BT_HCI_LE_INTERVAL_MIN´ (which is 6, corresponding to 7.5 ms).

Reply:
It was me that recommended @babrsn to use `BT_HCI_LE_INTERVAL_MIN` instead of `CONN_INTERVAL_MIN(x)` as the latter is intended for Connection Parameter Request and not the CONNECT_IND although the name `CONN_INTERVAL_MIN(x)` doesn't really suggest it. To prevent unintended behavior I would recommend that we introduce another macro if the CONNECT_IND also need to have proprietary interval support.

Reply:
> if the CONNECT_IND also need to have proprietary interval support.

This is not a requirement yet, hence outside the scope of this PR. We can take it up as separate PR as need be.